Mobl_ios_Callisto-master/MealTrackingPilot/LogCommonMeal/Views/OccasionPicker.swift

```diff
@@ -77,157 +62,132 @@ open class PickerView: UIView {
             }
         }
     }
-
+    
     /**
-        ScrollingStyle Enum.
-
-        - parameter Default: Show only the number of rows informed in data source.
-
-        - parameter Infinite: Loop through the data source offering a infinite scrolling experience to the user.
-    */
-
+     ScrollingStyle Enum.
+     
+     - parameter Default: Show only the number of items informed in data source.
+     
+     - parameter Infinite: Loop through the data source offering a infinite scrolling experience to the user.
+     */
+    
     @objc public enum ScrollingStyle: Int {
-        case `default`, infinite
+        case `default`
+        case infinite
     }
-
+    
+    /**
+     ScrollingDirection Enum.
+     
+     - parameter horizontal: Loads picker items with horizontal scrolling
+     
+     - parameter vertical: Loads picker items with vertical scrolling
+     */
+    
     @objc public enum ScrollingDirection: Int {
```

Comment:
Create a new file OccasionPicker+Helpers & move the enum to there

Reply:
Done